lookout/style/typos/typos_correction/utils.py

```diff
@@ -0,0 +1,172 @@
+import itertools
```

Comment:
I did that because maybe in future this corrector module will be used not only in lookout. It is quite an abstract toolkit for almost any type of data to correct.

Reply:
ok

Reply:
this actually doesn't change the fact that another module level is not required. I agree with @zurk, it seems unnecessary.

Reply:
Ok, it will be removed